docs/wave.rst

```diff
@@ -42,7 +43,7 @@ receiver coordinates are stored in an instance of :obj:`IShot()` called :obj:`da
   data.set_receiver_pos(receiver_coords)
   data.set_shape(nt, 101)
 
```

Comment:
Mistake of mine, started with 2D and never changed it for 3D. Supposed to be (20, 20, 20)